### PR TITLE
@okta/samples generator 1.15.0

### PR DESCRIPTION
**Pull request by jmelberg-okta**
Updates documentation to prepare for Dev Console rollout.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,16 +65,15 @@ After all the front-end dependencies install, you will need to install all compo
...
 
-Start the back-end for your sample application with `npm start` or ` php -S 0.0.0.0:3000 -t src/public `. This will start the app server on [http://localhost:3000](http://localhost:3000).
+Start the back-end for your sample application with `npm start` or `php -S 0.0.0.0:3000 -t src/public`. This will start the app server on [http://localhost:3000](http://localhost:3000).
 
 By default, this application uses a mock authorization server which responds to API requests like a configured Okta org - it's useful if you haven't yet set up OpenID Connect but would still like to try this sample. 
 
 To start the mock server, run the following in a second terminal window:
 ```bash
-# Starts the mock Okta server at http://127.0.0.1:7777
+# Starts the mock Okta server at http://127.0.0.01:7777
```
Comment (bretterer) [inline README.md]: Why did this get changed to 127.0.0.01?  it should remain 127.0.0.1

File: README.md [hunk 2]
```diff
@@ -83,7 +82,7 @@ If you'd like to test this sample against your own Okta org, follow [these steps
 // .samples.config.json
 {
   "oidc": {
-    "oktaUrl": "https://{{yourOktaOrg}}.oktapreview.com",
+    "oktaUrl": "https://{{yourOktaDomain}}.com",
```
Comment (bretterer) [inline README.md]: Why the change to remove `oktapreview`

File: README.md (hunk 1 above)
Comment (jmelberg-okta) [inline README.md]: Ah. That is a typo in our generator. Didn't notice it until just now. Good eye

File: README.md (hunk 2 above)
Comment (jmelberg-okta) [inline README.md]: New schema. Whenever we are referencing an organization, we are pointing toward `https://{yourOktaDomain}.com`. This will satisfy all of our org types.

Comment (bretterer) [inline README.md]: Shouldn't it be https://{yourOktaDomain} then, since we will be allowing custom domains, and i may not be using a `.com`?

Comment (jmelberg-okta) [inline README.md]: That is still on the roadmap. This new schema is now consistent throughout all of our samples + developer.okta.com.

Comment (bretterer) [inline README.md]: 👍

**Comment by jmelberg-okta** [comment]
@bretterer This is planned to roll out next week. Lets hold off on the merge until then.

File: README.md (hunk 2 above)
Comment (bretterer) [inline README.md]: I think we should call this {{orgUrl}} everywhere and remove `.com`... since on the new dashboard, it is referenced as `Org URL`
